package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "eslint-config-typescript",
+  "version": "1.0.0",
+  "description": "ESLint typescript sharable config for thesis wide projects",
+  "main": "index.js",
+  "repository": "https://github.com/thesis/eslint-config-typescript.git",
+  "author": "thesis",
```

Comment:
Ideally you'd use your name + thesis.co email here

Reply:
Done in 16e1f0c